Test/PesterLogger/RemoteLogCollector.Tests.ps1

```diff
@@ -297,4 +297,56 @@ Describe "RemoteLogCollector - with actual Testbeds" -Tags CI, Systest {
             Remove-AllContainers -Session $Sess1
         }
     }
+
+    Context "Windows Eventlog" {
+        BeforeEach {
+            Invoke-Command -Session $Sess1 {
+                New-EventLog -Source "Test" -LogName "TestLog"
+            }
+        }
+        AfterEach {
+            Invoke-Command -Session $Sess1 {
+                Remove-EventLog -LogName "TestLog"
+            }
+        }
+        It "reading from non-existing log collector returns proper error in logs" {
+            $Source = New-EventLogLogSource -Sessions $Sess1 -EventLogName "Invalid Name" -EventLogSource "Invalid source"
+            Initialize-PesterLogger -OutDir "TestDrive:\"
+
+            Merge-Logs -LogSources $Source
+
+            $ContentRaw = Get-Content -Raw "TestDrive:\*.Windows Eventlog.reading from non-existing log collector returns proper error in logs.txt"
+            $ContentRaw | Should -BeLike "*event log retrieval error:*"
+        }
+        It "writing and clearing works" {
```

Comment:
I'd split this test into two or three separate ones. As it: "gets messeges from eventlog", "ignores messeges from before initialization", "ignores messages from before clearcontent".